pontoon/base/static/js/translate.js

```diff
@@ -271,6 +282,24 @@ var Pontoon = (function (my) {
 
 
     /*
+     * Open batch editor
+     */
+    openBatchEditor: function () {
```

Comment:
Agreed. Also, there are many global variable we're assigning to Pontoon object and it's hard to keep track of them. We should spread them among such objects and maybe keep them in ObjectName.data instead of the top level, so they're easier to manage.